.github/workflows/build.yml

```diff
@@ -59,6 +59,24 @@ jobs:
           cmake -DBUILD_SHARED_LIBS=ON ..
           cmake --build . --config Release
 
+      - name: Get commit hash
+        id: commit
+        if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
+        uses: pr-mpt/actions-commit-hash@v2
+
+      - name: Pack artifacts
+        id: pack_artifacts
+        if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
+        run: |
+          zip rwkv-${{ env.BRANCH_NAME }}-${{ steps.commit.outputs.short }}-bin-linux.zip ./build/librwkv.so
+
+      - name: Upload artifacts
+        if: ${{ ( github.event_name == 'push' && github.ref == 'refs/heads/master' ) || github.event.inputs.create_release == 'true' }}
+        uses: actions/upload-artifact@v3
+        with:
+          path: |
+            rwkv-${{ env.BRANCH_NAME }}-${{ steps.commit.outputs.short }}-bin-linux.zip
```

Comment:
I think it needs to specify OS name, `ubuntu` in this case. As I know, there are many different Linuxes, and after building on Ubuntu library may just not work on some.

Is it possible to also include arch and OS version (for Mac OS too)? Maybe there is some env variable for that...

Reply:
Sure, I will handle it asap

Reply:
@saharNooby Hi, I have updated the file names. 
you could check the file names from https://github.com/yorkzero831/rwkv.cpp/releases/tag/master-a3fe1c6
ubuntu aseet will look like `rwkv-master-a3fe1c6-bin-Linux-Ubuntu-22.04-x86_64.zip`
macos asset will look like `rwkv-master-a3fe1c6-bin-Darwin-macOS-12.6.4-x86_64.zip`
I was thought macos will run on M1 which is arm64, seems github action dosen't support it yet

Reply:
Looks great! One small thing -- Windows zips have `rwkv.dll` in the zip root, without subdirectories. Can Linux/MacOS zips also have no subdirectories for consistency? Or will it break something?

Reply:
Oh I didn't realize it, will change, give me 10 mins

Reply:
@saharNooby it shuold be fixed you could verify from https://github.com/yorkzero831/rwkv.cpp/releases/tag/master-5662bf4